glnx-fdio.c

```diff
@@ -204,7 +203,7 @@ open_tmpfile_core (int dfd, const char *subpath,
    * link_tmpfile() below to rename the result after writing the file
    * in full. */
 #if defined(O_TMPFILE) && !defined(DISABLE_OTMPFILE) && !defined(ENABLE_WRPSEUDO_COMPAT)
-  fd = openat (dfd, subpath, O_TMPFILE|flags, mode);
+  glnx_autofd int fd = openat (dfd, subpath, O_TMPFILE|flags, mode);
```

Comment:
Minor, but let's scope this block so that we're not relying on `fd` being shadowed later below?

Reply:
Thanks, pushed with that change.